cmd/virt-launcher/virt-launcher.go

```diff
@@ -420,7 +417,7 @@ func main() {
 	// Marking Ready allows the container's readiness check to pass.
 	// This informs virt-controller that virt-launcher is ready to handle
 	// managing virtual machines.
-	markReady(*readinessFile)
+	markReady()
```

Comment:
rather than having this socket renaming and everything, why not simply start the command server here rather than earlier in the initialization process.

Reply:
The advantage is that we have exactly the same behaviour like before. We know that at this stage clients can connect since virt-launcher performed pings at this stage.